### ISSUE TITLE
Unable to run inference - can't load model on another path (windows)

### ISSUE THREAD

**Issue by RandomKaos999** [painebenjamin/app.enfugue.ai#131]
Hiya,
Just updated to 0.3.3 (on a windows 10 system), but ran into an issue of being unable to load any models at all on an alternate location. So I did it as a fresh install, renamed the old directory for the downloaded files, extracted to a new folder, cleared out C:\Users\<name>\.cache - this was deleted entirely. Browser cache also cleared for all time.

I can run the app, it fires up the UI, first time installation pops up and i'm ONLY changing the filepath for checkpoints to "F:\AIStuff\Models". I then attempt to execute a simple interface, no changes from default, just the prompt of "test".

Nothing happens apart from the below messages(s) - this is an example showing two different drives:

![image](https://github.com/painebenjamin/app.enfugue.ai/assets/138111560/447cd14b-a0de-4fdb-a2c9-6bfa9ae4f6f6)

With the path set as the original C directory - C:\Users\<name>\.cache\enfugue\checkpoints - I can download models and run inference. With the path set as anything else, it will download a model from civitai, then refuse to run it stating the above. That ghostmix model was downloaded as a test to 2 different drives. I was able to run "v1-5-pruned.ckpt" successfully after resetting the path to the C drive, downloading that model from the UI and doing the same "test" inference".

Just to let you know, there are zero log entries in the log file when attempting to this, it sits at 0kb in size, but no content in the file.
C:\Users\<name>\.cache\enfugue.log - Only other file in there is the .db - there is no engine log. The screenshot is all I can provide :(

TL;DR - other file paths appear to be broke on a Windows system at least, for models anyway. Works fine with other systems, Comfy etc.

**Comment by painebenjamin** [comment]
Oof. Definitely not correct. I had tried with different directories but not with completely different drives - and I was indeed able to recreate the issue. I've committed a fix in main and am building packages right now - I'll let you know when they're available for download. Sorry for the hassle!

**Comment by painebenjamin** [comment]
Didn't mean to close yet.

**Comment by painebenjamin** [comment]
Hi @RandomKaos999,

This is fixed. In my opinion, this was a breaking defect, so I re-released the `0.3.3` binaries with the fix, and you can update a conda environment with `pip install enfugue==0.3.3.post3`. Again, sorry for the hassle!

**Comment by RandomKaos999** [comment]
Tested, confirmed working. Thanks!

**Comment by RandomKaos999** [comment]
Hey again,
Hopefully you get this, it seems as though the "motion" installation option doesn't actually direct to the path properly during inference. You can set it in the installation path screen itself, you can list the motion modules when setting everything up under "More Model Configuration", but when actually running inference, the same thing occurs as with the models, kind of, apart from its not reference the true path (in my case of F:\AIStuff\motion). Assuming its the same/related, so commenting here.

![image](https://github.com/painebenjamin/app.enfugue.ai/assets/138111560/7870c9e5-7a5f-44d3-a864-73903afa3bd7)

I did originally try with v3 motion ckpt file, but thought it might not be supported or something, same thing occurs with v2. Same folder/drive as previous.

**Comment by RandomKaos999** [comment]
Doesn't seem to work if set back to the original path either, file(s) copied in there too. Assuming that i'm not daft, and its actually the motion option that signifies where the AnimateDiff modules go?

**Comment by painebenjamin** [comment]
Arg. Definitely related, but somewhere else down the line. Thanks for coming back and reporting again. Definitely need better testing around this. I'm digging again now.

**Comment by RandomKaos999** [comment]
Update, if it narrows it down, it works if you place the ckpt file direct into the software folder itself. Likely not using the path at all :)

**Comment by CodeAlexx** [comment]
I am a simple windows  users. I boot off a small C;\ ssd and use a bigger seperate disk to do my work. How do I change folder paths to the bigger one? Not all of us are savvy to understand your linux conventions to port to window structure.

**Comment by RandomKaos999** [comment]
Hi @CodeAlexx,
If you go Settings > Installation via the web interface. You will see all the paths there, its a case of clicking on the far right icon, "Change Directory", and then you need to either type in, or copy and paste the path into there. There is no browse for folder option.
